Makefile

```diff
@@ -230,7 +253,7 @@ WEBSITE_ASSETS_ALL = $(WEBSITE_ASSETS) $(addprefix catala-examples.tmp/,$(WEBSIT
 
 website-assets-base: build
 	$(call local_tmp_clone,catala-examples) && \
-	dune build $(addprefix _build/default/,$(WEBSITE_ASSETS_ALL))
+	dune build $(addprefix _build/default/,$(WEBSITE_ASSETS_ALL)) --profile=release
```

Comment:
Be careful, the absence of the release flag here may be intentional (I don't remember well what I did) due to https://github.com/CatalaLang/catala/issues/442.

Reply:
Actually it *was* there before my rewrite, but passed through the `DUNE_PROFILE` variable in the `website-assets` target, so normally that should be restoring the previous behaviour...

Reply:
If that's the case fine then :)